shared/utility/motion/splines/Footstep.cpp

```diff
@@ -85,15 +85,15 @@ namespace utility::motion::splines {
         stepFromSupport(tmpDiff);
     }
 
-    Eigen::Vector3f Footstep::poseAdd(const Eigen::Vector3f& pose, const Eigen::Vector3f& diff) const {
+    Eigen::Vector3f Footstep::poseAdd(const Eigen::Vector3f& pose, const Eigen::Vector3f& diff) {
```

Comment:
I would leave it up to personal preference in this case, but in general, I prefer to just have regular functions in the cpp file if there is a function that is static and it doesn't use any state from the class. Makes it clearer that it doesn't require any class state to function